src/styles/main.scss

```diff
@@ -0,0 +1,54 @@
+@import "./_variables.scss";
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.watch {
+    width: $width;
+    height: $height;
```

Comment:
you cant set `width` and `height` to BAM block

Reply:
agree, create BEM element for watch and move styles to element